dockerfile_scripts/setup_sh_env.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -x
+
+# This script is used to alter the default env for Bourne shells.
+# This is helpful since we install HPC related tools in HPC_DIR and
+# it would be good to have the ${HPC_DIR}/include be in the default
+# include path for gcc, etc.
+env_file=/etc/hpc-ai-env.sh
+echo "#!/bin/bash" >> $env_file
+
+hpc_cpath="export CPATH=${HPC_DIR}/include:${HPC_DIR}/include/linux:${HPC_DIR}/include/uapi"
```

Comment:
Do you need ":$CPATH" at the end of this?  I dont remember if CPATH is set by default or not, but it's being over-written here rather than pre-pended

Reply:
It wasn't set so far in our container but you're right we should probably prepend. I'll have to switch this because I'll need to expand variables such as HPC_DIR but *not* expand CPATH for the final version.